vsphere_plugin_common/__init__.py

```diff
@@ -1664,14 +1718,23 @@ def create_server(
         relospec.datastore = datastore.obj
         relospec.pool = resource_pool.obj
         if not auto_placement:
-            logger().warn(
+            self._logger.warn(
                 'Disabled autoplacement is not recomended for a cluster.'
             )
             relospec.host = host.obj
 
+        # get list nic mac's for remove
```

Comment:
Better phrased as: Get list of NIC MAC addresses for removal
(or: Get list of NIC MACs for removal)